policies/resource.yaml

```diff
@@ -104,3 +112,41 @@ resourcePolicy:
       condition:
         match:
           expr: request.resource.attr.aNumber <= 2
+
+    - actions:
+        - "relation-some"
+      effect: EFFECT_ALLOW
+      roles:
+        - USER
+      condition:
+        match:
+          expr: P.id in request.resource.attr.owners
```

Comment:
Thanks :)

That is the intended approach.

When you have a concrete instance of a resource the `owners` attributes would be a list of the principal IDs that own it (as you can fetch this from the DB eg `owners: ["user1", "user2"]`) and the usual condition matching would work.

When you need to list resources, it will return the predicate passed into Prisma via the adapter.

Reply:
Great then yeah this makes a lot of sense